pkg/apis/infinispan/v1/types_util.go

```diff
@@ -324,11 +324,13 @@ func (ispn *Infinispan) UpdateStatus(client client.Client, logger logr.Logger, s
 		if statusUpdate != nil {
 			statusUpdate()
 		}
-		err := client.Status().Update(context.TODO(), ispn)
+		ispn.Status.DeepCopyInto(&infinispan.Status)
```

Comment:
Why are we explicitly calling `DeepCopyInto` here? Can't we just do `infinispan.status = ispn.status` and the value will be copied.

Reply:
`DeepCopy` do the same if `struc` don't have an elements we need special operation, like arrays, slices, pointers, etc.

Reply:
And I don't want to be care about all of this

Reply:
ok by me